subjects/mobile-dev/chess/audit/README.md

```diff
@@ -1,19 +1,51 @@
+#### In order to run and hot reload the app either on emulator or device, follow the [instructions](https://docs.flutter.dev/get-started/test-drive?tab=androidstudio#run-the-app).
 
-###### Was the app committed within the required time period?
+###### Does the app run without crashing?
 
-###### Does the app run without crashes?
+##### Confirm if the app contains all the UI screens requested.
 
-###### Check the backend. When player is the waiting room and someone joins, does the game create UUID for the game session?
+- A Main menu
+- A Game board
+- A waiting room
 
-###### When 2 players are join game session, can a player with white pieces start the game?
+###### Does the app have all the UI screens requested?
 
-###### Try doing illegal moves, i.e. moving pawn 3 squares up, castling after moving the king, moving rook diagonally, etc. Does the app prevent such illegal moves?
+###### Does the app have a multiplayer mode for real-time matches which enables players to challenge and play against other?
+
+###### Can players invite each other to a game or join a public game?
+
+###### Does the app have a waiting room for players to wait for opponents and initiate game sessions?
+
+###### When two players join a game session, does the player with the white pieces play first?
+
+###### Does the app have a notification system to alert players when it's their turn to make a move?
+
+##### Try performing illegal moves (e.g., moving a pawn 3 squares up, castling after moving the king, moving a rook diagonally).
+
+###### Does the app prevent illegal moves?
 
 ###### After a move is played, does the other player receive the move?
 
-###### Play the game until one player is mated, does the game stop?
+##### Play the game until one player is mated.
+
+###### Does the game ends and the player who mated the other win?
+
+###### Does the backend generate a unique identifier (UUID) for each game session when a player joins the waiting room, ensuring proper identification and management of individual game sessions?
+
+#### Bonus
+
+###### +When the game reaches a terminal state (checkmate, stalemate, or draw), does the app send a message to both players indicating that the game is over?
+
+###### +After determining the winner of the game, does the app send a message to both players indicating which player has won?
+
+###### +Does the app allow players to restart the game?
+
+###### +Can players reset the board without restarting the entire game?
+
+###### +Is there a single-player story mode where players can engage in chess matches against AI opponents, progressing through a series of challenges or levels?
+
+###### +Are there AI-powered bots that players can play against in offline mode or when waiting for online opponents?
 
-###### Do chess moves come with appropriate Long algebraic notation?
+###### +Does the app have a mode where players can play against a computer AI with adjustable difficulty levels?
```

Comment:
I feel there is no difference between this question and the one above

Reply:
I left only the Bots bonus as it seems a better option to leave other than the"vs Computer" removed it from the readme as well